### DISCUSSION TITLE
Do I have to rely on the MacOS to develop iOS applications using MAUI?

### DISCUSSION THREAD

**Discussion by seabluescn** [dotnet/maui#14780] [General] [answered]
I have a PC with Windows 10 and an iPhone, can I use MAUI to develop and debug iOS applications? I don't need to publish the app to the Appstore, but I need the app to run on my iPhone, and I want to know if I must buy a MacBook?

**Comment by JohnHDev** [comment, marked as answer]
This should help: https://learn.microsoft.com/en-us/dotnet/maui/ios/hot-restart?view=net-maui-7.0

**Comment by Xyncgas** [comment]
Develop for the web, pray for apple drop dead
